client/me/concierge/reschedule/calendar-step.js

```diff
@@ -77,6 +78,24 @@ class CalendarStep extends Component {
 		}
 	}
 
+	renderDisallowed() {
```

Comment:
This looks the same as https://github.com/Automattic/wp-calypso/blob/trunk/client/me/concierge/cancel/index.js#L57. Let's dedupe them somehow :)

Reply:
I've made the requested change @southp, this is ready for review again.